### PR TITLE
dev/playerqueenview

### PR DESCRIPTION
**Pull request by Smile030504**

### CODE REVIEW

**Comment by Smile030504** [comment]
image

File: src/main/java/com/ouroboros/sleepingqueen/PreGameScreenController.java
Comment (Long2511) [inline src/main/java/com/ouroboros/sleepingqueen/PreGameScreenController.java:1]: Why Delete a function?

File: src/main/resources/com/ouroboros/sleepingqueen/view/board-view.fxml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <?import javafx.scene.layout.AnchorPane?>
 <?import javafx.scene.layout.HBox?>
 
-<AnchorPane prefHeight="720" prefWidth="1280" stylesheets="@../css/styles.css" xmlns="http://javafx.com/javafx/23.0.1" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.ouroboros.sleepingqueen.BoardViewController">
```
Comment (Long2511) [inline src/main/resources/com/ouroboros/sleepingqueen/view/board-view.fxml:6]: Why change the dimension and delete the styles.css

File: src/main/resources/com/ouroboros/sleepingqueen/view/rule-screen.fxml
Comment (Long2511) [inline src/main/resources/com/ouroboros/sleepingqueen/view/rule-screen.fxml:1]: No images in PR for review

**Review by Long2511 — CHANGES_REQUESTED** [review]

File: src/main/resources/com/ouroboros/sleepingqueen/view/player-score-and-menu.fxml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+         <children>
+             <fx:include id="player-score" source="player-score.fxml" />
+         </children>
+         <GridPane.margin>
+            <Insets left="40.0" top="20.0" />
+         </GridPane.margin>
+      </HBox>
+      <HBox prefHeight="100.0" prefWidth="200.0" GridPane.rowIndex="1">
+         <children>
+            <Button mnemonicParsing="false" text="menu">
```
Comment (Long2511) [inline src/main/resources/com/ouroboros/sleepingqueen/view/player-score-and-menu.fxml]: Should be "Menu" not "menu"

**Comment by Smile030504** [comment]
image

**Review by xiinthanh — APPROVED** [review]

**Review by Long2511 — APPROVED** [review]

**Review by xiinthanh — APPROVED** [review]

**Review by Long2511 — APPROVED** [review]

**Pull request merged**